MinPlatformPkg/Test/Library/TestPointCheckLib/MmCheckLoadedImage.c

```diff
@@ -0,0 +1,104 @@
+/** @file
+
```

Comment:
If this is a new file, can you please add a brief file description comment?

Reply:
Added.